code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -337,18 +337,24 @@ CONTENTS:
 
 	add_attack_logs(user, L, "Stunned with [src]")
 
-/obj/item/abductor_baton/proc/SleepAttack(mob/living/L,mob/living/user)
-	if(L.IsStunned() || L.IsSleeping())
-		L.visible_message("<span class='danger'>[user] has induced sleep in [L] with [src]!</span>", \
+/obj/item/abductor_baton/proc/SleepAttack(mob/living/L, mob/living/user)
+	var/mob/living/carbon/C = L
+	if(!iscarbon(L))
+		return
+	if(C.getStaminaLoss() < 100)
```

Comment:
Now you removed the sleeping check entirely, this should be
```suggestion
	if(C.getStaminaLoss() < 100 && !C.IsSleeping())
```

Reply:
Whoops. thanks for catching these.

Reply:
Upon testing, this logic makes it so stunning is not required for sleeping, for some god forsaken reason.

Reply:
fixed in latest, thank ya